### ISSUE TITLE
My swap to sell on secondary immediately wne to someone elses collection

### ISSUE THREAD

**Issue by snurpl** [hicetnunc2000/hicetnunc#1120]
When my swap for 25 xtz was appplied it immediately went to wallet tz1YtMN2US1JW3LdtmToaLdygyVaVfVhAPYE for 0 tez.
The swap is reflected in the history and the blockchain but it's not reflected on my hictory

This might've happened yesterday with a couple other swaps, I need to check.  Has this happened before?  Was my wallet being watched so this could be automated?  Can this be reversed?

**Comment by crzypatchwork** [comment]
you need to provide more information for us to investigate as your tz wallet and/or op hash of the transaction

**Comment by snurpl** [comment]
someone pointed out that the scroll wheel can change the amount if the cursor is in the area of the amount box.  I was using a trackpad and since the whole page doesn’t scroll down any further when I scroll to the swap button, the cursor stays in the amount area while I swiped down. This is what would’ve caused the amount to lower from 25 to zero.  I practiced this to check and that’s what happpens.  Not cool
